scala-package/examples/pom.xml

```diff
@@ -15,6 +15,7 @@
 
   <properties>
     <skipTests>true</skipTests>
+    <skipJavaTests>${skipTests}</skipJavaTests>
```

Comment:
What is this for? I don't think it is valid in maven to set one property based on another property and it is just set to false in modules which have java tests to run

Reply:
This keeps control of a Junit test is going to run or not, by default it is set to be false. In fact, this is working...